<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber_Dec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>32</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Hex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>32</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Hex</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>371</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dec">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Hex">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_MB">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_KB">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_GB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_GB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_KB">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>KB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MB">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MB</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
